snuba/migrations/snuba_migrations/generic_metrics/0002_sets_raw_table.py

```diff
@@ -0,0 +1,89 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import (
+    Array,
+    Column,
+    DateTime,
+    Float,
+    Nested,
+    String,
+    UInt,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+    local_table_name = "generic_metric_sets_raw_local"
+    dist_table_name = "generic_metric_sets_raw_dist"
+    columns: Sequence[Column[Modifiers]] = [
+        Column("use_case_id", String(Modifiers(low_cardinality=True))),
+        Column("org_id", UInt(64)),
+        Column("project_id", UInt(64)),
+        Column("metric_id", UInt(64)),
+        Column("timestamp", DateTime()),
+        Column("retention_days", UInt(16)),
+        Column(
+            "tags",
+            Nested(
+                [
+                    ("key", UInt(64)),
+                    ("indexed_value", UInt(64)),
+                    ("raw_value", String()),
+                ]
+            ),
+        ),
+        Column("set_values", Array(UInt(64))),
+        Column("count_value", Float(64)),
+        Column("distribution_values", Array(Float(64))),
+        Column("metric_type", String(Modifiers(low_cardinality=True))),
```

Comment:
The raw tables are specifically for sets. Then why do you need count_value and distribution_values here? Also there shouldn't be a need for metric_type as well. Could you provide some additional context?

Reply:
The thought was that we could replicate the exact same data schema for the raw table across clusters. So in theory we could merge without significant code changes if we find that two metric types have very low traffic, for example